packages/wrangler/e2e/helpers/run.ts

```diff
@@ -7,22 +7,41 @@ assert(
 	"Please provide a CLOUDFLARE_ACCOUNT_ID as an environment variable"
 );
 
-const CF_ID = `CLOUDFLARE_ACCOUNT_ID=${process.env.CLOUDFLARE_ACCOUNT_ID}`;
-const WRANGLER = process.env.WRANGLER ?? `npx wrangler@beta`;
-
-const RUN = `${CF_ID} ${WRANGLER}`;
+const RUN = process.env.WRANGLER ?? `npx --prefer-offline wrangler@beta`;
```

Comment:
Rather than hitting the beta deployed version, I think we could do:

```suggestion
const pathToWrangler = path.resolve(__dirname, "../../..");
const RUN = process.env.WRANGLER ?? `node "${pathToWrangler}"`;
```

And that might even avoid having to set the `WRANGLER` package in the CI job - and even avoid all that packaging and copying/moving in the job too.

Reply:
Or were you trying to be even more "pure" in terms of e2e?

Reply:
Yeah, the idea was that it should mimic running a package installed from npm